packages/fx-core/src/common/localSettingsConstants.ts

```diff
@@ -8,11 +8,11 @@ export const LocalSettingsTeamsAppKeys = Object.freeze({
 });
 
 export const LocalSettingsAuthKeys = Object.freeze({
-  AadClientId: "aadClientId",
-  AadClientSecret: "aadClientSecret",
-  AadObjectId: "aadObjectId",
-  AadOauth2PermissionScopeId: "aadOauth2PermissionScopeId",
-  AadApplicationIdUris: "aadApplicationIdUris",
+  AadClientId: "clientId",
```

Comment:
Just better style, align the key and value `ClientId: "clientId"`.

Reply:
Agree. updated in the latest commit.